snprc_ehr/resources/queries/study/BehaviorReport.sql

```diff
@@ -0,0 +1,19 @@
+-- Behavior Summary Reports
+SELECT year(b.ReportDate)                         AS ReportYear,
+       b.id                                         AS Id,
+       --b.NotificationNumber,
+       b.ReportDate,
+       --b.CloseDateTm,
+       b.Room                                       AS BehaviorLocation,
+       b.Behavior,
+       b.id.demographics.Gender                     AS Sex,
+       b.id.demographics.Species.Arc_species_code   As SpeciesCode,
+       b.id.demographics.Dam                        AS Dam,
+       b.id.demographics.rearing_type               AS RearingType,
+       b.id.demographics.species.arc_species_code.common_name AS Species,
+       b.id.curLocation.room                      AS CurrentRoom
```

Comment:
Should CurrentRoom be changed to the location at the time of the behavior incident?